api-reference/_hidden/GridBaseColumn/calculateCellValue.md

```diff
@@ -363,12 +361,12 @@ To invoke the default behavior, call the **this.defaultCalculateCellValue(rowDat
 
 ---
 
-[/note]
-
 #include uiwidgets-ref-functioncontext with { 
     value: "column's configuration"
 }
 
+[note] This function is called multiple times for every record: when the widget renders it, applies sorting or filtering, and computes summaries. We recommend that you keep calculations inside this function as simple as possible so as not to lower the widget's performance.
```

Comment:
```suggestion
[note] This function is called multiple times for every record: when the widget renders it, when it applies sorting or filtering, and when it computes summaries. It is recommend that you keep calculations inside this function as simple as possible to avoid hindering widget performance.
```

Reply:
> This function is called multiple times for every record: when the widget renders it, ...

Here, "it" refers to the record, while later:

> ... when it applies sorting or filtering, and when it computes summaries.

... "it" refers to the widget.

Reply:
can you rewrite it?

Reply:
```suggestion
[note] This function is called multiple times for every record: when the record is rendered, when sorting or filtering is applied, and when summaries are computed. It is recommend that you keep calculations inside this function as simple as possible to avoid hindering widget performance.
```